src/main/java/net/sf/jabref/collab/FileUpdateMonitor.java

```diff
@@ -194,7 +195,7 @@ public boolean copy() {
 
             boolean res = false;
             try {
-                res = FileUtil.copyFile(file, tmpFile.toFile(), true);
+                res = FileUtil.copyFile(Paths.get(file.toURI()), tmpFile, true);
```

Comment:
Since `tmpFile` is already a `Path` object, why don't you replace `File file` with `Path file`?